rotkehlchen/api/v1/fields.py

```diff
@@ -362,6 +363,36 @@ def _deserialize(
         return result
 
 
+class DBEnumField(fields.Field):
```

Comment:
No this is not needed. It's not any special "DBEnum" you are using. In order toachieve what you want was to do a change I already have in my branch. Overwrite `serialize()` in ChainID to use value.

```python
    def serialize(self) -> int:
        return self.value
```

Reply:
Aha, yeah, okay, got it :+1: